src/config.ts

```diff
@@ -0,0 +1,14 @@
+var argv = process.argv.slice(2);
+process.env.PORT = '5000';
```

Comment:
it rewrite any value you will set as environment variable. Better to write something like 
```
Number(process.env.PORT) || 5000
```

Reply:
okay, ths